pkg/annotation/aws.go

```diff
@@ -0,0 +1,77 @@
+package annotation
+
+//Keep the YAML documentation format as it is used to render in the CRD public documentation. You can use Markdown in the documentation field.
```

Comment:
@MarcelMue apparently there is a reason for these comments.

@paurosello will moving annotations to a different module here break CRD public documentation rendering? I guess it will.
Also, could we reformat somehow these comments? Feels a bit wrong to reply on basically invalid golang documentation formatting, AFAIK comment should start with var  / func / type name and then have some description.

Reply:
My disappointment is immeasurable, my day has been ruined. 

Fine to merge this and we deal with it later.

Reply:
![](https://media.giphy.com/media/40Z4EtasDsfMVEyX8H/giphy.gif)